test/functionalTest/cases/4330_onlyChangedAttrs_built_in_metadata/onlychangedattrs_not_working_with_built_in_metadata.test

```diff
@@ -0,0 +1,226 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+onlyChangedAttrrs not working with csubs cache
+
+--SHELL-INIT--
+dbDrop ${CB_DB_NAME} tk
+brokerStart CB 0 IPv4 -subCacheIval 2 -multiservice
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. Create entity E1 with attributes A and B
+# 02. Create subscription with onlyChangedAttrs true covering the entity
+# 03. Update E1-A
+# 04. Sleep until cache refres, update E1-A again
+# 05. Dump accumulator and see only A atribute in both notifications
+#
+
+
+echo '01. Create entity E1 with attributes A and B'
+echo '============================================'
+payload='{
+   "id": "E1",
+   "type": "T",
+   "A": {
+     "value": "foo",
+     "type": "Text"
+   },
+   "B": {
+     "value": 123,
+     "type": "Number"
+   }
+}'
+orionCurl --url /v2/entities --payload "$payload" --tenant tk --servicePath /
```

Comment:
```suggestion
orionCurl --url /v2/entities --payload "$payload"
```

To keep test simpler, I'd suggest to remove tenant and servicepath. Also in `dbDrop` statements

Reply:
`-multiservice` in brokerStart line can be also dropped

Reply:
Note: Removing tenant involves to change request headers expectations

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4335/commits/44d62c3b3c6a68ac7cd4aacff02bcf8ccb388c9c